diagnostic.md

```diff
@@ -98,11 +132,26 @@ JSON.
 ```
 
 ```sh
-# your answer here
+curl --include --request "PATCH" "http://www.example.com/donuts/42" \
+  --header "Content-Type: application/json" \
+  --data ${
+    \"donut\": {
+      \"name\": \"Krüller\",
+    }
+  }
 ```
 
 Write an AJAX request to change the donut on the server using JSON.
 
 ```js
-let changeDonut = /* your answer here */;
+let changeDonut = function ()
+                    {
+                    $.ajax({
+                    url: 'http://www.example.com/donuts/42' + data.donut.id,
+                    method: 'PATCH',
+                    data: data
```

Comment:
Same with this. You don't know what data is because it's not defined. OR you have to create an object.